### PR TITLE
Fixes saving of certain policy data

### PR DESCRIPTION
**Pull request by dev-gh**
Fixes saving and processing of:
- counter for TLS errors
- device id
- message codes

### CODE REVIEW

**Comment by dev-gh** [comment]
@AGaliuzov @VProdanov @AByzhynar @OHerasym 
Please review

File: src/components/policy/include/policy/policy_table/types.h [hunk 1]
```diff
@@ -201,13 +201,13 @@ struct ModuleConfig : CompositeType {
...
-               const ServiceEndpoints& endpoints,
-               const NumberOfNotificationsPerMinute&
-                   notifications_per_minute_by_priority);
+			   int64_t exchange_after_x_kilometers,
+			   uint8_t exchange_after_x_days,
+			   uint16_t timeout_after_x_seconds,
+			   const SecondsBetweenRetries& seconds_between_retries,
+			   const ServiceEndpoints& endpoints,
+			   const NumberOfNotificationsPerMinute&
+				   notifications_per_minute_by_priority);
```
Comment (AByzhynar) [inline src/components/policy/include/policy/policy_table/types.h]: @dev-gh Strange formatting. Don't you think so?

File: src/components/policy/include/policy/policy_table/types.h [hunk 2]
```diff
@@ -327,20 +327,20 @@ struct AppLevel : CompositeType {
...
+		   uint16_t minutes_in_hmi_none,
+		   uint16_t count_of_user_selections,
+		   uint16_t count_of_rejections_sync_out_of_memory,
+		   uint16_t count_of_rejections_nickname_mismatch,
+		   uint16_t count_of_rejections_duplicate_name,
+		   uint16_t count_of_rejected_rpc_calls,
+		   uint16_t count_of_rpcs_sent_in_hmi_none,
+		   uint16_t count_of_removals_for_bad_behavior,
+		   uint16_t count_of_tls_errors,
+		   uint16_t count_of_run_attempts_while_revoked);
```
Comment (AByzhynar) [inline src/components/policy/include/policy/policy_table/types.h]: @dev-gh Strange formatting. Don't you think so?

Comment (dev-gh) [inline src/components/policy/include/policy/policy_table/types.h]: @AByzhynar 
Changed with a0fa829

File: src/components/policy/include/policy/policy_table/types.h (hunk 1 above)
Comment (dev-gh) [inline src/components/policy/include/policy/policy_table/types.h]: Changed with a0fa829

**Comment by dev-gh** [comment]
@AGaliuzov @VProdanov @OHerasym
Gentle reminder

**Comment by OHerasym** [comment]
Reviewed

**Comment by VProdanov** [comment]
Reviewed

**Comment by AByzhynar** [comment]
Reviewed

**Comment by dev-gh** [comment]
@AGaliuzov 
Please review

**Pull request merged**